### DISCUSSION TITLE
Use of Clip-seg for vid2vid masking?

### DISCUSSION THREAD

**Discussion by MichaelPawluk** [volotat/SD-CN-Animation#44] [Ideas]
Great script so far. The results are on par or better than other animation solutions. I am no clear on what the implementation would look like or even be possible but could an image segmentation tool like clip-seg be utilized as a way to select and mask components of the video for the img2img generation?

**Comment by volotat** [comment]
For now I could only say maybe as I'm not well familiar with how clip-seg works but I totally see how it could be useful.
